### PR TITLE
show status message when navigating hunks

### LINKED ISSUE #237: Display warning message when trying to go to next/previous hunk if there isn't one
I think it would be a good idea to display a warning message on the status line saying that there are no more hunks after/before the last/first one (of course, only when the wrap option is set to false). Other popular git sign packages like gitgutter have this functionality. What are your thoughts on this?

### PR DESCRIPTION
**Pull request by zapling**
Closes #237

### CODE REVIEW

**Comment by mroavi** [comment]
Hey @zapling ! Thanks for implementing this! I have a small suggestion. Would it be possible to print "No more hunks" in red color?

**Comment by zapling** [comment]
@mroavi  I added the highlight group `WarningMsg` so it now looks like this

Screenshot_2021-07-03_13-53-51

**Comment by mroavi** [comment]
Awesome. Thanks. I just tried it out and it is working great.

**Comment by mroavi** [comment]
One last thing that might be useful is an option to enable it or disable it. I can imagine that there are people out there who might not want this.

**Comment by zapling** [comment]
Sounds responsible

**Comment by lewis6991** [comment]
Is there not a default vim option similar to `wrapscan` that we can piggy back off for default behaviour? For displaying results of the `/` command, if `'shortmess'` contains `S`, then results are not shown.

**Comment by lewis6991** [comment]
LGTM

**Comment by zapling** [comment]
I'll just update the docs

File: types/types.d.tl [hunk 1]
```diff
@@ -191,6 +191,7 @@ global record vim
     diffopt: string
     wrapscan: boolean
     updatetime: number
+    shortmess: string
```
Comment (zapling) [inline types/types.d.tl:194]: I just added this value manually, is there another way?

Comment (lewis6991) [reply types/types.d.tl:194]: Nope, It's all manual. We just add to it as we need.

**Pull request merged**